readthedocs/projects/tasks.py

```diff
@@ -424,6 +427,14 @@ def build_docs_class(self, builder_class):
         builder.move()
         return success
 
+    def send_notifications(self):
+        """Send notifications on build failure
+
+        Don't send failure notices on ``stable`` version builds.
```

Comment:
Not sure why we decided not to send mail on stable builds -- can you think of a good reason? I think it was to stop spamming, but we should probably enable it.